examples/icmp_socket.py

```diff
@@ -0,0 +1,68 @@
+from rflx.pyrflx import (MessageValue, Package, PyRFLX)
```

Comment:
This file is not checked by the linters. To change that add the `examples` directory to `python-packages` in the `Makefile`: `python-packages := bin examples rflx tests tools stubs`.

Please also check the use of blank lines in this file.

Reply:
fixed and squashed into 6d14bb9